app/src/main/java/org/glucosio/android/db/DatabaseHandler.java

```diff
@@ -667,9 +663,9 @@ public void deleteKetoneReading(KetoneReading reading) {
         return idArray;
     }
 
-    public ArrayList<Double> getKetoneReadingAsArray() {
+    public List<Double> getKetoneReadingAsArray() {
```

Comment:
The method name does not reflect the return type.

Reply:
maybe ` getKetoneReadingAsList` or `getKetoneReadingList`?

Reply:
Also renamed